src/wallet/rpcwallet.cpp

```diff
@@ -2823,6 +2824,9 @@ static UniValue listunspent(const JSONRPCRequest& request)
             "    \"scriptPubKey\" : \"key\",   (string) the script key\n"
             "    \"amount\" : x.xxx,         (numeric) the transaction output amount in " + CURRENCY_UNIT + "\n"
             "    \"confirmations\" : n,      (numeric) The number of confirmations\n"
+            "    \"ancestorcount\" : n,      (numeric) The number of unconfirmed ancestors if spent\n"
+            "    \"ancestorsize\" : n,       (numeric) The virtual transaction size of all unconfirmed ancestors\n"
+            "    \"ancestorfees\" : n,       (numeric) The modified fees of all unconfirmed ancestors\n"
```

Comment:
They are only present (optional) when unconfirmed? Should mention that, or make then unconditional.

Reply:
Fixed